src/unix/mod.rs

```diff
@@ -143,6 +143,9 @@ cfg_if! {
     } else if #[cfg(target_os = "emscripten")] {
         #[link(name = "c")]
         extern {}
+    } else if #[cfg(all(target_vendor = "rumprun", target_os = "netbsd"))] {
+        #[link(name = "m")]
```

Comment:
We pass a few flags to the linker to say that default system libraries shouldn't be included by default, so shouldn't we link both libc and libm like the block just below this one?